src/blitter/40bpp_anim.cpp

```diff
@@ -234,18 +235,14 @@ inline void Blitter_40bppAnim::Draw(const Blitter::BlitterParams *bp, ZoomLevel
 					break;
 
 				case BM_TRANSPARENT:
-					/* TODO -- We make an assumption here that the remap in fact is transparency, not some colour.
-					 *  This is never a problem with the code we produce, but newgrfs can make it fail... or at least:
-					 *  we produce a result the newgrf maker didn't expect ;) */
-
 					/* Make the current colour a bit more black, so it looks like this image is transparent */
 					src_n += n;
 					if (src_px->a == 255) {
 						src_px += n;
 						do {
 							/* If the anim buffer contains a color value, the image composition will
-							 * only look at the RGB brightness value. As such, we can simply darken the
-							 * RGB value to darken the anim color. */
+							* only look at the RGB brightness value. As such, we can simply darken the
+							* RGB value to darken the anim color. */
```

Comment:
Your editor broke the sanity again! :P

Reply:
We need a commit-checker to test for "\t\* "...!